### ISSUE TITLE
junction window is slow

### ISSUE THREAD

**Issue by symphorien** [NixOS/nixpkgs#197908]
### Describe the bug
running `re.sonny.Junction` from `nix-shell -p junction` displays a transparent window which is only drawn after 40 seconds (I timed it, it's not an exaggeration). Similarly, clicking the menu button takes about 20s to display the menu. Only the close button is "fast".

### Steps To Reproduce
1. run `re.sonny.Junction`
2. it's slow
3. ...

### Expected behavior
it displays fast

### Screenshots
on the right is the transparent window and my wallpaper.
![image](https://user-images.githubusercontent.com/12595971/198041626-d6662822-2ce5-4beb-bfb8-805d9b8c8a49.png)

### Additional context
I'm not running gnome, but xfce+i3+picom if it's relevant. But as junction eventually works, something is probably not missing.

### Notify maintainers
cc @hqurve 

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.74, NixOS, 22.11 (Raccoon), 22.11pre420601.b39fd6e4ede`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - channels(root): `"nixos, nixos-unstable"`
 - channels(symphorien): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by wamserma** [comment]
Opens within a few seconds here with LXQt on a twelve-year old CPU... 
tested with: `/nix/store/fiml1gm3cqh6ym8yqd6jph72gvlyvqj3-junction-1.5.0/bin/re.sonny.Junction`

**Comment by hqurve** [comment]
I'm using plasma on unstable and I experience normal app delay. (1-2 seconds). `/nix/store/b58zd02jvalgx9ab0w13qgbg36fxzsii-junction-1.5.0/bin/re.sonny.Junction` (Maybe one or two iterations old). 

There is https://github.com/sonnyp/Junction/issues/45; however, I do not know how applicable it is. I tried to follow the steps using a shell based on the derivation (plus blueprint-compiler) but I encounter the following error

```
❯ time ./re.sonny.Junction --terminate_after_init

(re.sonny.Junction:184189): Gjs-WARNING **: 22:07:47.547: JS ERROR: TypeError: source is null
_init@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:705:1
@resource:///re/sonny/Junction/common.js:3:25
@file:///home/joshua/kde/src/Junction/re.sonny.Junction:49:6

```
(will try again tomorrow)

PS: Also thanks, I didnt realize that I needed to update junction to 1.6 (I'll try to do it soon™, .... but its a javascript dependency)
